packages/stream_video/CHANGELOG.md

```
@@ -1,6 +1,20 @@
-## Unreleased
+## 1.0.0
+
+### 🚧 Breaking changes
+
+#### CallKit/Ringing
+In this release, we removed the dependency on `flutter_callkit_incoming`, which introduces breaking changes in the CallKit and ringing functionality:
+
+* **API renames and type changes**
+    - `onCallKitEvent` is now `onRingingEvent`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)